wherehows-web/app/templates/components/nacho/dropdown/hover-dropdown.hbs

```diff
@@ -1,20 +1,28 @@
-{{#basic-dropdown as |bd|}}
-  {{#bd.trigger class="nacho-drop-down__trigger"
-                onMouseDown=(action "prevent")
-                onMouseEnter=(action "showDropDown")
-                onMouseLeave=(action "hideDropDown")}}
-    <strong class="nacho-drop-down__active">
-      {{selectedDropDown.label}}
-    </strong>
+{{#basic-dropdown renderInPlace=@renderInPlace as |bd|}}
+  {{#if (not-eq @wrappedTriggerComponent true)}}
+    {{#bd.trigger class="nacho-drop-down__trigger"
+                  onMouseDown=(action "prevent")
+                  onMouseEnter=(action "showDropDown")
+                  onMouseLeave=(action "hideDropDown")}}
+      <strong class="nacho-drop-down__active">
+        {{selectedDropDown.label}}
+      </strong>
 
-    {{#if selectedDropDown}}
-      <span class="nacho-drop-down__active__toggle">
-        {{fa-icon (if isExpanded "caret-up" "caret-down")}}
-      </span>
-    {{/if}}
-  {{/bd.trigger}}
+      {{#if selectedDropDown}}
+        <span class="nacho-drop-down__active__toggle">
+          {{fa-icon (if isExpanded "caret-up" "caret-down")}}
+        </span>
+      {{/if}}
+    {{/bd.trigger}}
+  {{/if}}
 
   {{yield (hash
+            trigger=(component bd.trigger
+              onMouseDown=(action "prevent")
```

Comment:
I think following the standard/approach set in the documentation allows everyone to coalesce on one approach rather than personal preference or bike-shedding
See examples in ember.js guides:
https://guides.emberjs.com/release/components/triggering-changes-with-actions/
```hbs
{{button-with-confirmation
  text="Click to send your message."
  onConfirm=(action "sendMessage")
}}
```